src/mesh_sampling.cpp

```diff
@@ -41,6 +43,19 @@ void create_cloud(const aiScene * scene, unsigned N, const bfs::path & out_path,
   {
     success = mesh_sampling::io::saveQhullFile(out, *cloud);
   }
+  else if(extension == ".stl")
+  {
+    pcl::ConvexHull<PointT> convex{};
```

Comment:
I'm not familiar with PCL convex hull API, but couldn't we use it to generate the convex hull files we need in `mc_rtc` directly?
This would simplify and speed up the robot description generation scripts, and also could have the added benefit of making it easy to provide an API to generate the convex hulls on the fly from any robot model (or pointcloud data as @TsuruMasato would love to have).

Reply:
Yes. I think there's enough data to build an `sch::S_Polyhedron` directly but that would be a bit more work. If this is something @TsuruMasato needs he can contact me and we can work on converting PCL point clouds to objects that can be used for collision avoidance for mc_rtc